sharing_configs/admin.py

```diff
@@ -98,8 +103,8 @@ def sharing_configs_export_view(self, request, object_id):
                 self.message_user(request, msg, level=messages.ERROR)
             return render(
                 request,
-                self.import_template,
-                {"form": form, "opts": self.model._meta},
+                self.change_form_export_template,
+                {"form": form, "context": context, "opts": self.model._meta},
```

Comment:
I think `context` var name is confusing here. This whole dict is already a context. So you have a context with a nested 'context' var. Could you rename it or add 'main_url' to the context without additional nesting?

Reply:
Done